WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -4471,6 +4478,8 @@ function WeakAuras.NewAura(sourceData, regionType, targetId)
     WeakAuras.NewDisplayButton(data);
     WeakAuras.PickAndEditDisplay(data.id);
   end
+  ViragDevTool_AddData(data, data.id .. " data")
```

Comment:
just a note that it has to be removed later, still useful for testing

Reply:
lul i forgot to remove that, thanks